pkg/inference/backends/llamacpp/download.go

```diff
@@ -27,12 +27,22 @@ const (
 var (
 	ShouldUseGPUVariant     bool
 	ShouldUseGPUVariantLock sync.Mutex
+	ShouldUpdateServer      = true
+	ShouldUpdateServerLock  sync.Mutex
 	errLlamaCppUpToDate     = errors.New("bundled llama.cpp version is up to date, no need to update")
```

Comment:
Not really related to this PR, but maybe we should replace `errLlamaCppUpToDate` with a `return nil` as well and a log entry (like you've done with the `update disabled` case).  I worry it's sort of confusing in the status output.